### ISSUE TITLE
Error C2065 occurs in surface_on_nurbs

### ISSUE THREAD

**Issue by UnaNancyOwen** [PointCloudLibrary/pcl#1493]
The error C2065 occurs in surface_on_nurbs when build PCL master/HEAD(e97b664) on Visual C++.
(This error occurs only when enable the BUILD/surface_on_nurbs option in CMake.)

The error message is as follows.
```
error C2065: 'M_PI' : undeclared identifier	C:\pcl\surface\src\on_nurbs\fitting_sphere_pdm.cpp		196
error C2065: 'M_PI' : undeclared identifier	C:\pcl\surface\src\on_nurbs\fitting_curve_pdm.cpp		230
error C2065: 'M_PI' : undeclared identifier	C:\pcl\surface\src\on_nurbs\fitting_curve_2d_pdm.cpp	423
error C2065: 'M_PI' : undeclared identifier	C:\pcl\surface\src\on_nurbs\fitting_curve_pdm.cpp		267
error C2065: 'M_PI' : undeclared identifier	C:\pcl\surface\src\on_nurbs\fitting_cylinder_pdm.cpp	259
error C2065: 'M_PI' : undeclared identifier	C:\pcl\surface\src\on_nurbs\fitting_cylinder_pdm.cpp	318
error C2065: 'M_PI' : undeclared identifier	C:\pcl\surface\src\on_nurbs\fitting_curve_2d_apdm.cpp	562
```

**Comment by UnaNancyOwen** [comment]
I think should be fix by adding either of the following.
Do you have any good ideas?

* idea 1 
```
#define _USE_MATH_DEFINES
#include <cmath>
```

* idea 2 
```

#ifndef M_PI
    #define M_PI 3.14159265358979323846f
#endif
```

**Comment by VictorLamoine** [comment]
There is already [pcl_macros.h](https://github.com/PointCloudLibrary/pcl/blob/master/common/include/pcl/pcl_macros.h) but the `#define` looks unusual.

**Comment by UnaNancyOwen** [comment]
Thank you @VictorLamoine 
Fixed by adding the "#include <pcl/pcl_macros.h>" to these source code.
I sent a pull request #1495.

**Comment by taketwo** [comment]
Fixed via #1495.
